eof/download.py

```diff
@@ -89,35 +87,36 @@ def download_eofs(
     # First, check that Scihub isn't having issues
     if not force_asf:
         client = DataspaceClient(username=cdse_user, password=cdse_password)
-        # try to search on scihub
-        if sentinel_file:
-            query = client.query_orbit_for_product(sentinel_file, orbit_type=orbit_type)
-        else:
-            query = client.query_orbit_by_dt(orbit_dts, missions, orbit_type=orbit_type)
-
-        if query:
-            logger.info("Attempting download from SciHub")
-            results = client.download_all(query, output_directory=save_dir)
-            filenames.extend(results)
-            dataspace_successful = True
+        if client._username and client._password:
```

Comment:
Please pay attention, in the ASFClient you are using `self.username` and `self.password` without underscores.